src/mail/editor/MailEditor.ts

```diff
@@ -198,7 +198,8 @@ export class MailEditor implements Component<MailEditorAttrs> {
 		})
 
 		this.editor.initialized.promise.then(() => {
-			this.inlineImageElements = replaceCidsWithInlineImages(this.editor.getDOM(), model.loadedInlineImages, (cid, event, dom) => {
+			const dom = this.editor.getDOM()
```

Comment:
looks a bit odd to me, why don't we just pass shadow root into it instead?

Reply:
because we'll forget the next time we add a usage of replaceCidsWithInlineImages

Reply:
Ah, okay, the idea is that the helper function works on any *DOM element* and not on any *node*. It also makes sense.